actors/builtin/miner/policy.go

```diff
@@ -81,47 +91,67 @@ var MaxSealDuration = map[abi.RegisteredSealProof]abi.ChainEpoch{
 	abi.RegisteredSealProof_StackedDrg64GiBV1:  abi.ChainEpoch(10000),
 }
```

Comment:
**Change requested**: There is an FIP where decided to split this into two variables: https://github.com/filecoin-project/specs-actors/issues/672